tasks/main.yml

```diff
@@ -28,8 +37,54 @@
     user_install: false
   with_items: "{{ fluentd_plugins }}"
 
+- name: retrieve initial ExecStart
+  shell: 'grep -i "ExecStart=.*/.*" /lib/systemd/system/multi-user.target.wants/td-agent.service | sed -r "s/^([^\/])*(\/.*)/\2/"'
```

Comment:
Is it possible to use `/lib/systemd/system/td-agent.service`? We should use the original service file

Reply:
Sure, copy/paste mistake